core/Makefile

```diff
@@ -56,6 +56,10 @@ dev: gen
 	audius-compose down
 	audius-compose up db core core-content-1 core-content-2 core-content-3 eth-ganache
 
+.PHONY: test
+test: gen
```

Comment:
I don't have strong opinions yet, maybe hold off on putting `dev` in here since we might prefer to iterate on the tests themselves with `make test` rather than reset the whole stack, but we might change our minds.